projects/ion/src/lib/chip-group/chip-group.component.html

```diff
@@ -6,6 +6,7 @@
     [disabled]="chip.disabled || disabled"
     [selected]="chip.selected"
     [showDropdown]="chip.selected && chip.options"
+    [showToggle]="true"
```

Comment:
Shouldn't this property be optional?

Reply:
I don't think so, because it wont add any wanted behavior to the chip group, it was only added so it could make the chip with dropdown work as expected. But if we have a case I can set this as a variable property.